zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -141,7 +146,7 @@ class ObjectManagementFacade(
         )
 
         try {
-            logger.trace { "Creating object $objectRequest" }
+            logger.info { "Creating object $objectRequest" }
```

Comment:
I think we shouldn't want to log the whole `objectRequest` as info here because it might be large, complex and contain sensitive data.

Reply:
Is this a useful thing to log? I think this should be covered by the plugin action.

Reply:
I propose to remove the logger line here